src/coreclr/debug/dbgutil/elfreader.cpp

```diff
@@ -127,7 +127,7 @@ ElfReader::PopulateForSymbolLookup(uint64_t baseAddress)
     // Enumerate program headers searching for the PT_DYNAMIC header, etc.
     if (!EnumerateProgramHeaders(
         baseAddress,
-#if defined(TARGET_LINUX_MUSL) || defined(TARGET_RISCV64)
+#if defined(TARGET_LINUX_MUSL) || defined(TARGET_RISCV64) || defined(HOST_ANDROID)
         // On musl based platforms (Alpine) and RISCV64 (VisionFive2 board),
         // the below dynamic entries for hash,
```

Comment:
```suggestion
        // On some platforms, the below dynamic entries for hash,
```
We do not need to replicate the ifdef condition in the comment. It just gets out of sync.